okio/src/jvmMain/kotlin/okio/JvmSystemFileSystem.kt

```diff
@@ -123,6 +124,10 @@ internal open class JvmSystemFileSystem : FileSystem() {
   }
 
   override fun delete(path: Path, mustExist: Boolean) {
+    if (Thread.currentThread().isInterrupted) {
```

Comment:
Should be `interrupted()`, not `isInterrupted`.

Reply:
ae4d73fe1d05ee0ad53f7ab2469ebba9fea20603